src/main/kotlin/org/rust/lang/core/psi/impl/RustRawStringLiteralImpl.kt

```diff
@@ -4,6 +4,15 @@ import com.intellij.psi.tree.IElementType
 import org.rust.lang.core.psi.RustLiteralTokenType
 
 class RustRawStringLiteralImpl(type: IElementType, text: CharSequence) : RustTextLiteralImplBase(type, text) {
+    private var _hashes: Int = 0
+
+    val hashes: Int
+        get() {
+            @Suppress("UNUSED_VARIABLE")
+            val o = offsets // ensure #computeOffsets() has been called
```

Comment:
This is probably temporary hack, I'm thinking yet on how to implement this as computing offsets may fail in this case.

The problem is, we are fed with highlighter iterator output, which is polluted with escape sequence tokens, so for `b"foo\nba<caret>r"` we won't get `<BYTE_STRING_LITERAL>(b"foo\\nbar")`, but `<BYTE_STRING_LITERAL>(bar")`.